src/components/detail/detail.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Detail from ".";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+
+test("renders children", () => {
+  render(<Detail>foo</Detail>);
+
+  expect(screen.getByText("foo")).toBeVisible();
+});
+
+test("renders with provided `footnote`", () => {
+  render(<Detail footnote="extra info">foo</Detail>);
+
+  expect(screen.getByText("extra info")).toBeVisible();
```

Comment:
**suggestion** (non-blocking): as you've already added a `data-role` for the footnote, you could add an extra assertion to verify the text is contained within the footnote element. But it's not essential as you're still checking the text is rendered here. I'm happy with either approach